ovgs.proto

```diff
@@ -1,175 +1,241 @@
 // Copyright (c) 2023 Arista Networks, Inc.
 // Use of this source code is governed by the BSD 2-Clause License
 // that can be found in the LICENSE file.
-
 syntax = "proto3";
 
-import "google/protobuf/timestamp.proto";
-
 package ovgs.v1;
 
-option go_package = "arista/sztp/ovgs/gen";
+import "google/protobuf/timestamp.proto";
+
+option go_package = "github.com/aristanetworks/ownership-voucher-grpc/ovgs";
 
-enum userRole {
-    UNSPEC = 0;
-    SUPPORT = 1;
-    ADMIN = 2;
-    REQUESTOR = 3;
+enum UserRole {
+  USER_ROLE_UNSPECIFIED = 0;
+  USER_ROLE_SUPPORT = 1;
+  USER_ROLE_ADMIN = 2;
+  USER_ROLE_REQUESTOR = 3;
 }
 
-message user {
-    string username = 1;     // username or service account name
-    string user_type = 2;    // "account" for service accounts and "user" for user accounts
-    string org_id = 3;       // org_id = "org-" + org name
-    userRole user_role = 4;  // role of the user, either ADMIN or REQUESTOR
+enum AccountType {
+  ACCOUNT_TYPE_UNSPECIFIED = 0;
+  ACCOUNT_TYPE_USER = 1;
+  ACCOUNT_TYPE_SERVICE_ACCOUNT = 2;
+}
+  
+message User {
+  string username = 1;     // username or service account name
+  AccountType user_type = 2;  // user account type
+  string org_id = 3;       // org_id = "org-" + org name
+  UserRole user_role = 4;  // role of the user, either ADMIN or REQUESTOR
 }
 
-message createGroupRequest {
-    string parent = 1;             // parent group ID, it could be org ID or a group ID
-    string group_description = 2;  // general description of the group
+message CreateGroupRequest {
+  // parent group ID, it could be org ID or a group ID
+  string parent = 1;
+  // general description of the group             
+  string group_description = 2;  
 }
 
-message createGroupResponse {
-    string group_id = 1;
+message CreateGroupResponse {
+  string group_id = 1;
 }
 
-message viewGroupRequest {
-    string group_id = 1;
+message GetGroupRequest {
+  string group_id = 1;
 }
 
-message viewGroupResponse {
-    repeated string certs = 1;
-    repeated string switch_serials = 2;
-    repeated user users = 3;
-    repeated string children = 4;
+message GetGroupResponse {
+  // list of certificate names in the group.
+  repeated string certs = 1;
+  // list of device serial numbers in the group.
+  repeated string device_serials = 2;
+  // list of users in the group.
+  repeated User users = 3;
+  // list of group ids that are children of this group.
+  repeated string children = 4;
```

Comment:
It was mentioned that the user friendly group description (as compared to the group_id, which the format = group-<uuid> isn't echoed back in the viewGroupResponse.

Since you're already refactoring the message types, could you please add a string field to the GetGroupResponse message? This is what is specified in CreateGroupRequest.

Reply:
fixed